Gruntfile.js

```diff
@@ -383,6 +383,23 @@ module.exports = function (grunt) {
             src: ['**', '!hmda-pilot.zip']
           }
         ]
+      },
+      'codedeploy': {
```

Comment:
What is the difference between the compress:codedeploy task and the compress:hmda-pilot task above? Do we really need both?